doc/api/zlib.md

```diff
@@ -308,7 +311,7 @@ ignored by the decompression classes.
 * `level` {integer} (compression only)
 * `memLevel` {integer} (compression only)
 * `strategy` {integer} (compression only)
-* `dictionary` {Buffer|TypedArray|DataView} (deflate/inflate only, empty dictionary by
+* `dictionary` {Buffer|TypedArray|DataView|ArrayBuffer} (deflate/inflate only, empty dictionary by
```

Comment:
Looks like this line might exceed 80 characters... either you can re-wrap, or we can do it while landing this.